deploy/crds/kieapp.crd.yaml

```diff
@@ -86,6 +86,10 @@ spec:
                         description: The JMX ObjectName of the JaasSecurityDomain
                           used to decrypt the password.
                         type: string
+                      loginModule:
+                        description: A flag to set login module to optional. The default
+                          value is required
+                        type: string
```

Comment:
to make the field as enum you need to do as it is done here:
https://github.com/kiegroup/kie-cloud-operator/blob/master/pkg/apis/app/v2/kieapp_types.go#L21

set the var type and on its comments, add the enum valid values.

@tchughesiv  was it that you have suggested?

Reply:
@spolti I added the enum here but I really forgot to update the type in crd.  https://github.com/kiegroup/kie-cloud-operator/commit/448a87c671a4c54170ed54142d028342bee8a118#diff-4d98202026f495310fe4aa8f64f71057eb9fa83104da7480469153b7d6fe810bR360

Reply:
I guess it could be something like this:

```
                      searchScope:
                        description: SearchScopeType Type used to define how the LDAP
                          searches are performed
                        enum:
                        - SUBTREE_SCOPE
                        - OBJECT_SCOPE
                        - ONELEVEL_SCOPE
                        type: string
```

Reply:
the crd is automatically updated, no manual changes are required there.